packages/ui/src/stories/Forms/NumberFormFIeld.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from "@storybook/react";
```

Comment:
is the name a typo? ``NumberFormF**i**eld``?

Reply:
Ugh. Yes.

On macOS, if I make a typo like this, it's sometimes hard to fix those (renames from VS Code don't work, renames from command line do work, but it's easy to reopen the old file in VS Code because it caches the name, and then it rewrites it back, and then if you've committed the file you have to do `git mv` instead of `mv`).

Reply:
Ah yea, I've had that happen before, definitely a pain.